src/utils/storage.js

```diff
@@ -0,0 +1,108 @@
+import AsyncStorage from '@react-native-community/async-storage';
+
+import { DEFAULT_USER } from './constants';
+import { getLemma } from '../api/api';
+
+const DEFAULT_USER_ERROR = 'No user selected! Have you logged in?';
+
+export const getUser = async () => {
+  try {
+    const user = await AsyncStorage.getItem('@user');
+    if (user !== null) {
+      return user;
+    }
+    return DEFAULT_USER;
+  } catch (e) {
+    return DEFAULT_USER;
+  }
+};
+
+export const setUser = async (user) => {
+  try {
+    await AsyncStorage.setItem('@user', user);
+    return true;
+  } catch {
+    return false;
+  }
+};
+
+export const getUserData = async (userParam) => {
+  let user;
+  if (!userParam) user = await getUser();
+  else user = userParam;
```

Comment:
Minor style note but while this syntax _works_, it's really uncommon to see if/else blocks without brackets/indentation.

I _think_ that you can make this shorter with `const user = userParam || await getUser()` but there may be a quirk with using `await` that way I'm not aware of...